apps/backend/src/user/user.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@nestjs/common'
+import { PrismaService } from 'src/prisma/prisma.service'
+import { Prisma, User } from '../../node_modules/.prisma/client'
+
+@Injectable()
+export class UserService {
+  constructor(private prisma: PrismaService) {}
+
+  async findAll(): Promise<User[]> {
+    return this.prisma.user.findMany()
```

Comment:
To tak, chciałbym, żebyśmy mieli pełny zestaw user.controller user.service i user.repository tak jak planowalismy robic w CleenR. Z odpowiednimi testami. To ma posluzyc za przyklad dla kolejnych endpointow.

controller - HTTP/nest
service - kod biznesowy
repository - prisma